Cubical/Foundations/RelationalStructure.agda

```diff
@@ -270,14 +254,16 @@ structuredQER→structuredEquiv {ρ = ρ} θ (X , s) (Y , t) R r .rel =
             Trunc.rec
               (squash/ _ _)
               (λ {(x , px , r) →
-                cong (E.Thm .fst) (sym px)
-                ∙ eq/ (R .snd .fwd x) y (R .snd .zigzag (R .snd .bwdRel y) r (R .snd .fwdRel x))
+                (cong (E.Thm .fst) (sym px)
+                ∙ E.relToFwd≡ r)
                 ∙ py})
               qr}))
-        (elimProp
-          {B = λ qx →
-            E.Thm .fst qx ≡ qy → [R] .fst qx qy}
+        (elimProp {B = λ qx → E.Thm .fst qx ≡ qy → [R] .fst qx qy}
```

Comment:
Can't you use elimProp2?

Reply:
Not without generalizing it, elimProp2 takes (x y : A / R) but the two arguments here are from different set quotients.